docs/reference/search/more-like-this.asciidoc

```diff
@@ -1,32 +0,0 @@
-[[search-more-like-this]]
```

Comment:
I think you also need to remove the include from `docs/reference/search.asciidoc` for the docs to build?